### PR TITLE
spvm: Format all disks before the test starts

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

File: lib/bootloader_pvm.pm [hunk 1]
```diff
@@ -117,10 +117,15 @@ sub prepare_pvm_installation {
 
     # Delete partition table before starting installation
     select_console('install-shell');
-    foreach my $disk (split(',', get_var('DISK_DEVICES', 'sda'))) {
```
Comment (rwx788) [inline lib/bootloader_pvm.pm]: As we get rid of this variable, let's remove it from `variables.md` too.

Comment (Soulofdestiny) [reply lib/bootloader_pvm.pm]: yep, but also found better solution =)

File: lib/bootloader_pvm.pm [hunk 2]
```diff
@@ -117,10 +117,13 @@ sub prepare_pvm_installation {
...
-        script_run("wipefs -a /dev/$disk");
-        # For cryptlvm+activate existing scenario, create empty enrypted partition
-        create_encrypted_part("$disk") if get_var('ENCRYPT_ACTIVATE_EXISTING');
+    
+
+    my $disks = script_output('lsblk -n -p -l -o NAME -d -e 7,11');
```
Comment (rwx788) [inline lib/bootloader_pvm.pm]: I remember issues with script_output being polluted, but I guess in the beginning of the installation we should not see them.

File: lib/bootloader_pvm.pm [hunk 3]
```diff
@@ -117,10 +117,13 @@ sub prepare_pvm_installation {
...
     select_console('install-shell');
-    foreach my $disk (split(',', get_var('DISK_DEVICES', 'sda'))) {
-        script_run("wipefs -a /dev/$disk");
-        # For cryptlvm+activate existing scenario, create empty enrypted partition
-        create_encrypted_part("$disk") if get_var('ENCRYPT_ACTIVATE_EXISTING');
+
+
+    my $disks = script_output('lsblk -n -p -l -o NAME -d -e 7,11');
+    for my $d (split('\n', $disks)) {
+        script_run("echo 'wipefs -a $d'");
```
Comment (rwx788) [inline lib/bootloader_pvm.pm]: Let's remove this line too ;)

File: lib/bootloader_pvm.pm [hunk 4]
```diff
@@ -117,10 +117,12 @@ sub prepare_pvm_installation {
...
-    foreach my $disk (split(',', get_var('DISK_DEVICES', 'sda'))) {
-        script_run("wipefs -a /dev/$disk");
-        # For cryptlvm+activate existing scenario, create empty enrypted partition
-        create_encrypted_part("$disk") if get_var('ENCRYPT_ACTIVATE_EXISTING');
+
+
+    my $disks = script_output('lsblk -n -p -l -o NAME -d -e 7,11');
+    for my $d (split('\n', $disks)) {
+        script_run "wipefs -a $d";
+        create_encrypted_part("$d") if get_var('ENCRYPT_ACTIVATE_EXISTING');
```
Comment (JRivrain) [inline lib/bootloader_pvm.pm]: Hi, this caused s little issue here https://openqa.suse.de/tests/3909752#step/bootloader_start/36 as then in create_encrypted_part we have  `assert_script_run "parted -s /dev/$disk mklabel gpt";`

**Pull request merged**